### ISSUE TITLE
Dúvida multithreading

### ISSUE THREAD

**Issue by gonzaleZMMR** [premium-minds/summer-internship-exercise-2021#3]
Boa tarde. Tenho uma dúvida relativamente à implementação de multithreading.

Visto que não tenho experiência na utilização do Future, fui pesquisar para saber um pouco acerca do assunto e para encontrar uma solução para o problema.

Uma possível implementação será executar uma ou mais tarefas num future e usar um ExecutorService com newFixedThreadPool de tamanho [numberOfWorkers]? Ou há alguma outra implementação que eu me esteja a esquecer?

**Comment by amadoandre** [comment]
Sim, a tua possível implementação é valida. Existem outros tipo de implementações que podem também ser consideradas validas. 
Cabe-te a ti perceber se queres ir por esse caminho ou investigar outros.
